presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -15,6 +15,18 @@ services:
     volumes:
       - ../../../../:/docker/volumes/presto
 
+  presto-master-ldap:
```

Comment:
Why do you need to add this section to the `standard.yml`? As i understand it is being used in `singlenode-ldap` profile only. Why not just move this section there?